### PR TITLE
[control-plane-manager] added creating a backup before a potentially dangerous action in docs

### PR DESCRIPTION
**Pull request by alexey-gavrilov-flant**
## Description
Added creating a backup before a potentially dangerous action in docs.

## Why do we need it in the patch release (if we do)?

Not necessary.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: fix 
summary: Added creating a backup before a potentially dangerous action.
impact_level: low 
```

### CODE REVIEW

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ title: "Управление control plane: FAQ"
 >
 > Важно иметь нечётное количество мастеров для обеспечения кворума.
 
+1. Делаем [бэкап](faq.html#резервное-копирование-etcd-и-восстановление) `etcd` и `/etc/kubernetes`
+1. Переносим архив на сервер вне кластера (на локальную машину)
+1. Нужно убедится что в кластере нет [алертов](..300-prometheus/faq.html#как-получить-информацию-об-алертах-в-кластере) которые помешают создать новые master
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:21]:
```suggestion
1. Сделайте [резервную копию](faq.html#резервное-копирование-etcd-и-восстановление) `etcd` и папки `/etc/kubernetes`.
1. Скопируйте полученный архив на пределы кластера (например, на локальную машину).
1. Убедитесь, что в кластере нет [алертов](../300-prometheus/faq.html#как-получить-информацию-об-алертах-в-кластере), которые помешают создать новые master-узел.
```

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 2]
```diff
@@ -48,6 +51,9 @@ title: "Управление control plane: FAQ"
 
 ## Как уменьшить число master-узлов в облачном кластере (multi-master в single-master)?
 
+1. Делаем [бэкап](faq.html#резервное-копирование-etcd-и-восстановление) `etcd` и `/etc/kubernetes`
+1. Переносим архив на сервер вне кластера (на локальную машину)
+1. Нужно убедится что в кластере нет [алертов](..300-prometheus/faq.html#как-получить-информацию-об-алертах-в-кластере) которые помешают создать новые master
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:56]:
```suggestion
1. Сделайте [резервную копию `etcd`](faq.html#резервное-копирование-etcd-и-восстановление) и папки `/etc/kubernetes`.
1. Скопируйте полученный архив за пределы кластера (например, на локальную машину).
1. Убедитесь, что в кластере нет [алертов](../300-prometheus/faq.html#как-получить-информацию-об-алертах-в-кластере), которые помешают создать новые master-узел.
```

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 3]
```diff
@@ -112,6 +118,9 @@ title: "Управление control plane: FAQ"
 
 ## Как убрать роль master-узла, сохранив узел?
 
+1. Делаем [бэкап](faq.html#резервное-копирование-etcd-и-восстановление) `etcd` и `/etc/kubernetes`
+1. Переносим архив на сервер вне кластера (на локальную машину)
+1. Нужно убедится что в кластере нет [алертов](..300-prometheus/faq.html#как-получить-информацию-об-алертах-в-кластере) которые помешают создать новые master
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:123]:
```suggestion
1. Сделайте [резервную копию `etcd`](faq.html#резервное-копирование-etcd-и-восстановление) и папки `/etc/kubernetes`.
1. Скопируйте полученный архив за пределы кластера (например, на локальную машину).
1. Убедитесь, что в кластере нет [алертов](../300-prometheus/faq.html#как-получить-информацию-об-алертах-в-кластере), которые помешают создать новые master-узлы.
```

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 4]
```diff
@@ -136,6 +145,9 @@ title: "Управление control plane: FAQ"
 
 ## Как изменить образ ОС в multi-master-кластере?
 
+1. Делаем [бэкап](faq.html#резервное-копирование-etcd-и-восстановление) `etcd` и `/etc/kubernetes`
+1. Переносим архив на сервер вне кластера (на локальную машину)
+1. Нужно убедится что в кластере нет [алертов](..300-prometheus/faq.html#как-получить-информацию-об-алертах-в-кластере) которые помешают создать новые master
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:150]:
```suggestion
1. Сделайте [резервную копию `etcd`](faq.html#резервное-копирование-etcd-и-восстановление) и папки `/etc/kubernetes`.
1. Скопируйте полученный архив за пределы кластера (например, на локальную машину).
1. Убедитесь, что в кластере нет [алертов](../300-prometheus/faq.html#как-получить-информацию-об-алертах-в-кластере), которые помешают обновить master-узлы.
```

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 5]
```diff
@@ -16,6 +16,9 @@ Adding a master node to a static or hybrid cluster has no difference from adding
 >
 > It is important to have an odd number of masters to ensure a quorum.
 
+1. Make a [backup](faq.html#etc-backup-and-restore) of `etcd` and `/etc/kubernetes`
+1. Transfer the archive to a server outside the cluster (on a local machine)
+1. Make sure there are no [alerts](..300-prometheus/faq.html#how-to-get-information-about-alerts-in-a-cluster) in the cluster that will prevent new masters from being created
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ.md:21]:
```suggestion
1. Make a [backup of `etcd`](faq.html#etc-backup-and-restore) and the `/etc/kubernetes` directory.
1. Transfer the archive to a server outside the cluster (e.g., on a local machine).
1. Ensure there are no [alerts](../300-prometheus/faq.html#how-to-get-information-about-alerts-in-a-cluster) in the cluster that can prevent the creation of new master nodes.
```

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 6]
```diff
@@ -48,6 +51,9 @@ Adding a master node to a static or hybrid cluster has no difference from adding
 
 ## How do I reduce the number of master nodes in a cloud cluster (multi-master to single-master)?
 
+1. Make a [backup](faq.html#etc-backup-and-restore) of `etcd` and `/etc/kubernetes`
+1. Transfer the archive to a server outside the cluster (on a local machine)
+1. Make sure there are no [alerts](..300-prometheus/faq.html#how-to-get-information-about-alerts-in-a-cluster) in the cluster that will prevent new masters from being created
 1. Run the appropriate edition and version of the Deckhouse installer container **on the local machine** (change the container registry address if necessary):
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ.md:57]:
```suggestion
1. Make a [backup of `etcd`](faq.html#etc-backup-and-restore) and the `/etc/kubernetes` directory.
1. Transfer the archive to a server outside the cluster (e.g., on a local machine).
1. Ensure there are no [alerts](../300-prometheus/faq.html#how-to-get-information-about-alerts-in-a-cluster) in the cluster that can prevent the update of the master nodes.
1. Run the appropriate edition and version of the Deckhouse installer container **on the local machine** (change the container registry address if necessary):
```

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 7]
```diff
@@ -112,6 +118,9 @@ Adding a master node to a static or hybrid cluster has no difference from adding
 
 ## How do I dismiss the master role while keeping the node?
 
+1. Make a [backup](faq.html#etc-backup-and-restore) of `etcd` and `/etc/kubernetes`
+1. Transfer the archive to a server outside the cluster (on a local machine)
+1. Make sure there are no [alerts](..300-prometheus/faq.html#how-to-get-information-about-alerts-in-a-cluster) in the cluster that will prevent new masters from being created
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ.md:123]:
```suggestion
1. Make a [backup of `etcd`](faq.html#etc-backup-and-restore) and the `/etc/kubernetes` directory.
1. Transfer the archive to a server outside the cluster (e.g., on a local machine).
1. Ensure there are no [alerts](../300-prometheus/faq.html#how-to-get-information-about-alerts-in-a-cluster) in the cluster that can prevent the update of the master nodes.
```

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 8]
```diff
@@ -136,6 +145,9 @@ Adding a master node to a static or hybrid cluster has no difference from adding
 
 ## How do I switch to a different OS image in a multi-master cluster?
 
+1. Make a [backup](faq.html#etc-backup-and-restore) of `etcd` and `/etc/kubernetes`
+1. Transfer the archive to a server outside the cluster (on a local machine)
+1. Make sure there are no [alerts](..300-prometheus/faq.html#how-to-get-information-about-alerts-in-a-cluster) in the cluster that will prevent new masters from being created
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ.md:150]:
```suggestion
1. Make a [backup of `etcd`](faq.html#etc-backup-and-restore) and the `/etc/kubernetes` directory.
1. Transfer the archive to a server outside the cluster (e.g., on a local machine).
1. Ensure there are no [alerts](../300-prometheus/faq.html#how-to-get-information-about-alerts-in-a-cluster) in the cluster that can prevent the update of the master nodes.
```

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [5433](https://github.com/deckhouse/deckhouse/pull/5433) to the branch [release-1.49](https://github.com/deckhouse/deckhouse/tree/release-1.49) successful!

**Pull request merged**